### PR TITLE
Add To Wishlist functionality 

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Add To Wishlist Widget Added
- Added wishlist management functionality using API Call.
 WISHLIST => addToWishlist, RemoveFromWishlist.
- Update nav bar Badges when added or removed.
- Add & Remove cards From the wishlist page & Cart page as well.
- Refactor some of the Code.

image

### CODE REVIEW

File: Zero-Gravity-Store-React/src/Components/Navigation/Navigation.css [hunk 1]
```diff
@@ -198,11 +198,12 @@
     background-color: #ff0000;
     padding: 6px;
     border-radius: 50%;
-    top: -7px;
+    top: -13px;
```
Comment (ranitasahaa1230) [inline Zero-Gravity-Store-React/src/Components/Navigation/Navigation.css:201]: Instead of px, you can use rem or % also

**Review by ranitasahaa1230 — APPROVED** [review]
Use destructuring wherever required.
Good naming convention followed
Good use of map wherever required.
 Rest looks good to me

File: Zero-Gravity-Store-React/src/Contexts/Wishlist-context.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              { headers: { authorization: authState.token } }
+            );
+            wishlistDispatch({
+              type: "ADD_TO_WISHLIST",
+              payload: res.data.wishlist,
+            });
+            console.log(res.data.wishlist);
+          } catch (error) {
+            alert(error);
+            console.log(product);
```
Comment (anupkgurung) [inline Zero-Gravity-Store-React/src/Contexts/Wishlist-context.js:33]: remove logs.

File: Zero-Gravity-Store-React/src/Pages/Wishlist/Components/WishlistProducts.js [hunk 3]
```diff
@@ -1,92 +1,69 @@
-export function WishlistProducts(){
-    return (
+import axios from "axios";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../../Contexts/Auth-context";
+import { useCart } from "../../../Contexts/Cart-context";
+import { useWishlist } from "../../../Contexts/Wishlist-context";
```
Comment (anupkgurung) [inline Zero-Gravity-Store-React/src/Pages/Wishlist/Components/WishlistProducts.js:5]: use barrel exports

**Review by anupkgurung — APPROVED** [review]
nice component structure.
you can use toast instead of  alert.
remove console log

File: Zero-Gravity-Store-React/src/Pages/Wishlist/Components/WishlistProducts.js (hunk 3 above)
Comment (jeetbhalani15) [reply Zero-Gravity-Store-React/src/Pages/Wishlist/Components/WishlistProducts.js:5]: ok gonna try it

**Pull request merged**